src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,130 @@
+// Default component properties, defined for :host. Properties which can not
```

Comment:
If the card is active and has a badge, the badge should be on top of the border (see figma). now it's behind the border.

Reply:
Until yesterday the badge was behind the border, I'm 100% sure about it because to solve it Federico and I introduced the z-index system with :before and :after. 
I'll proceed to change it again.. 😠

Reply:
Oh no :(